extensions/sql-database-projects/src/tools/buildHelper.ts

```diff
@@ -51,29 +59,30 @@ export class BuildHelper {
 			await fs.mkdir(this.extensionBuildDir);
 		}
 
-		const buildfilesPath = await this.getBuildDirPathFromMssqlTools();
-		buildFiles.forEach(async (fileName) => {
+		// download the Microsoft.Build.Sql sdk nuget
+		const httpClient = new HttpClient();
+		const nugetPath = path.join(this.extensionBuildDir, `${fullSdkName}.nupkg`);
+		await httpClient.download(microsoftBuildSqlUrl, nugetPath, outputChannel);
+
+		// extract the files from the nuget
+		await extractZip(nugetPath, { dir: path.join(this.extensionDir, buildDirectory, sdkName) });
+
+		outputChannel?.appendLine(constants.extractingDacFxDlls);
```

Comment:
should we output the log message prior to extracting the ZIP above so the user can see what is currently in progress (for example if extraction hangs/takes a long time for some reason).